CHANGES_UNRELEASED.md

```diff
@@ -2,4 +2,24 @@
 
 # Unreleased Changes
 
+<<<<<<< HEAD
```

Comment:
Sorry, looks like you need to resolve this conflict still (Let us know if you need help with this)